src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -553,6 +550,8 @@ public void testAllCheckSections() throws Exception {
                                     .isTrue();
                 }
 
+                validateCheckSection(moduleFactory, fileName, sectionName, section);
```

Comment:
This and moduleFactory is just a revert of 81448ea434cda883b748d3d2c77b2f58a8b22bbe